README.md

```diff
@@ -1,33 +1,34 @@
 # Build
 
-This project contains common build scripts that are used by a number of open source projects at Upbound. We standardize on a makefile interface to all projects with simple verbs. We attempt to put more logic in makefiles and closer to the project and less on side of CI. This enables the builds to run on multiple environments including laptops and different CI/CD systems. Most build and test operations are designed to run in a container that contains all the tools necessary.
+This project - commonly known as "the build submodule" -  is Crossplane's
+Makefile library. This library is used to build `crossplane/crossplane`,
+`crossplane/crossplane-runtime`, and most Crossplane providers.
 
-Some features include:
- * support for multiple platforms and architectures
- * support for Linux and macOS hosts
- * support for build channels for releases
- * support for releasing artifacts
- * support for golang, helm, kubernetes, nodejs, yarn, s3, and docker
+The build submodule is only intended to support Crossplane. There's no guarantee
+of stability. There are no releases or versions, and any commit may introduce a
+breaking change. Third parties are recommended to fork rather than taking a
+dependency.
```

Comment:
no guarantee of stability and breaking changes and this being used by all providers sounds scary 😅 or do providers count as third party?

Reply:
There's already no guarantee, this is just being honest about it.